src/lib/LED/devRGB.cpp

```diff
@@ -18,9 +31,18 @@ void WS281Binit()
 
 void WS281BsetLED(uint32_t color)
 {
-    strip.ClearTo(RgbColor(HtmlColor(color)));
+    strip.SetPixelColor(STATUS_LED_NUMBER, RgbColor(HtmlColor(color)));
     strip.Show();
 }
+
+void WS281BsetStripColour(RgbColor color)
```

Comment:
Why this uses different interface than ´WS281BsetLED()` function? 
Now this requires to make RgbColor class public. It would be better to keep that local and only inside this file and use `uint32_t` argument instead and convert value `RgbColor(HtmlColor(color))`... 

Could you open the use case a bit more?

Reply:
Missed that 😬